### DISCUSSION TITLE
为什么已经挂载了但是在本地看不到？

### DISCUSSION THREAD

**Discussion by Piggy-ch** [AlistGo/alist#6876] [Q&A]
通过Alist桌面端挂载到本地，但是在本地看不到这个盘，在浏览器上是能够正常访问的
![image](https://github.com/user-attachments/assets/e875c106-4bda-47b4-912f-96c611d27394)
![image](https://github.com/user-attachments/assets/714c50be-388d-4ee1-9310-625b68c9de6b)

**Comment by anwen-anyi** [comment]
F:

**Reply by wgy5231** [reply]
我有冒号也不行

**Comment by wgy5231** [comment]
同
